kimchi/src/verifier_index.rs

```diff
@@ -131,6 +135,10 @@ pub struct VerifierIndex<G: KimchiCurve> {
     /// The mapping between powers of alpha and constraints
     #[serde(skip)]
     pub powers_of_alpha: Alphas<G::ScalarField>,
+
+    // Foreign field modulus
+    #[serde(skip)]
```

Comment:
Mmmm good point, this may have been brought from a previous version and then I never went back to it again. I guess we would like to serialize this as well, I will change it.